example_docs/docs/conf.py

```diff
@@ -76,96 +43,55 @@
     "nbsphinx",
 ]
 
-# -----------------------------------------------------------------------------
-# Autodoc
-# -----------------------------------------------------------------------------
-
-# Keep aligned with Terra.
-autosummary_generate = True
-autosummary_generate_overwrite = False
-autoclass_content = "both"
-autodoc_typehints = "description"
-autodoc_typehints_description_target = "documented_params"
-
-
-# If true, figures, tables and code-blocks are automatically numbered if they
-# have a caption.
-numfig = True
-
-# A dictionary mapping 'figure', 'table', 'code-block' and 'section' to
-# strings that are used for format of figure numbers. As a special character,
-# %s will be replaced to figure number.
-numfig_format = {
-    'table': 'Table %s'
-}
-# The language for content autogenerated by Sphinx. Refer to documentation
-# for a list of supported languages.
-#
-# This is also used if you do content translation via gettext catalogs.
-# Usually you set "language" from the command line for these cases.
-language = "en"
-
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-# This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = ['_build', '**.ipynb_checkpoints']
-
-# The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'colorful'
-
-# A boolean that decides whether module names are prepended to all object names
-# (for object types where a “module” of some kind is defined), e.g. for
-# py:function directives.
-add_module_names = False
-
-# A list of prefixes that are ignored for sorting the Python module index
-# (e.g., if this is set to ['foo.'], then foo.bar is shown under B, not F).
-# This can be handy if you document a project that consists of a single
-# package. Works only for the HTML builder currently.
-modindex_common_prefix = ['qiskit.']
-
-# -- Configuration for extlinks extension ------------------------------------
-# Refer to https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html
-
-
-# -- Options for HTML output -------------------------------------------------
-
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
-
-#html_sidebars = {'**': ['globaltoc.html']}
 html_last_updated_fmt = '%Y/%m/%d'
 
+html_theme = 'qiskit_sphinx_theme'
 html_theme_options = {
     'logo_only': True,
     'display_version': True,
     'prev_next_buttons_location': 'bottom',
 }
-
-# qiskit package specific variables
 html_context = {
-    # toggle analytics
+    # Add "Was this page useful?" to the footer.
```

Comment:
so this is technically true specifically for this example docs `conf.py` but in production (i.e. for pages live at qiskit.org domains) this also tracks data relating to page views. I'm not sure if we really need to specify the difference in functionality in the comment here or not 🤔 I guess we've probably explained it enough in the README, but I get nervous sometimes about ensuring we're being as transparent as possible about data/analytics tracking. I'd say it would be a nice-to-have to explicitly mention the analytics/data tracking element of this here, but it's not blocking the PR

Reply:
I can fix this in a follow up PR :)